v19.1/managed-authorization.md

```diff
@@ -1,22 +1,36 @@
 ---
-title: User Management
-summary: Learn how to create SQL users.
+title: Authorization
+summary: Learn about the authorization features for Managed CockroachDB clusters.
 toc: true
 build_for: [managed]
 ---
 
+Managed CockroachDB supports network authorization and user authorization.
+
+## Network authorization
+
+Managed CockroachDB requires you to authorize the networks that can access the cluster. This helps prevent denial-of-service and brute force password attacks.
+
+Authorize your application server’s network and your local machine’s network by adding the IP addresses in the CIDR notation using the [Networking page](managed-connect-to-your-cluster.html#authorize-your-network). If you change your location, you will need to authorize the new location’s network, else the connection from that network will be rejected.
+
+{{site.data.alerts.callout_info}}
+While developing and testing your application, you may whitelist `0.0.0.0/0`, which allows all networks. However, before moving into production, make sure you delete the `0.0.0.0/0` network since it allows anybody who uses your password to access the database and makes your application vulnerable to security breaches.
```

Comment:
"vulnerable to security breaches". That's a bit dramatic. I think the main concern currently is as mentioned above: denial-of-service (easy) and brute-force-attacks (not easy, but also turns into a DOS)

Reply:
Hmm..in that case, can I stop at "access the database" and remove the "makes your application..." part?